pyproject.toml

```diff
@@ -34,6 +34,7 @@ dev = [
     "pyright==1.1.345",
 ]
 gui = ["textual==0.76.0", "pyclip==0.7"]
+jax = ["jax==0.4.31"]
```

Comment:
We will be relying on unstable things in the API, hence the minor version pin.

Reply:
That does not at all sound like an upcoming nightmare :smile: